lib/testing/init.fnl

```diff
@@ -0,0 +1,116 @@
+(var suites [])
+(var state {:suite nil
+            :before []
+            :after []
+            :ran 0
+            :failed 0
+            :passed 0})
+
+;; 4-bit colors
+;; https://i.stack.imgur.com/9UVnC.png
+
+(local colors {:red   "31"
+               :green "92"})
+
+(fn describe
+  [suite-name suite-f]
+  (table.insert suites {:name suite-name
+                        :suite suite-f
+                        :before []
+                        :after []
+                        :tests []}))
+
+(fn it
+  [description test-f]
+  (if state.suite
```

Comment:
maybe we should raise some error if an `it` is defined outside of a `describe`, otherwise someone could write tests that are skipped without noticing.

Reply:
I'll give that a shot.